src/Arcus.spec.ts

```diff
@@ -64,14 +67,35 @@ describe('Arcus', () => {
     expect(netflix).toMatchSnapshot();
   });
 
-  it('getTransactionsList', async () => {
-    const transactions = await arcus.getTransactions();
+  describe('getTransactions', () => {
+    it('gets Transaction without parameters', async () => {
+      const transactions = await arcus.getTransactions();
 
-    expect(transactions).toHaveLengthWithin(95, 105);
+      expect(transactions).toHaveLengthWithin(95, 105);
 
-    const first = transactions[0];
+      const first = transactions[0];
 
-    expect(first).toMatchSnapshot();
+      expect(first).toMatchSnapshot();
```

Comment:
This tests is probably very flacky, and it might break as soon as we have a new transaction, right?

Reply:
Ill change tests to not use snapshots as they dont seem right for dynamic data, althought this wouldn't break as is the first transaction that has been made what im looking at.